setup.py

```diff
@@ -25,13 +25,13 @@
     "nose-focus",
     "colorama",
     "numpy",
-    "scipy>=0.16.1,<1.0.0",
     "flake8",
     "yanc",
     "remotecv",
-    "pyssim",
+    "pyssim>=0.4.0",
     "cairosvg>=1.0.0,<2.0.0,!=1.0.21",
     "preggy>=1.3.0",
+    "opencv-python",
```

Comment:
Here we're making this a mandatory dependency, which is not the case for Thumbor, I believe. Only if the user decides to use [such engine](/thumbor/opencv-engine) and [configure Thumbor to do so](http://thumbor.readthedocs.io/en/latest/configuration.html#engine) and in such cases he/she should install the desired engine. Perhaps docs regarding this should be improved.

Reply:
This is a list of test dependencies. opencv could be used in PIL engine to convert 16bit TIFFs to png.

Reply:
ah, it's also used in redeye filter and detectors. And in order to test that it's all required.